README.md

```diff
@@ -72,26 +78,26 @@ To run all the tests, use the test script.
 
 ```sh
 # from the root of the project
-$ ./tests/start_tests.bash
+$ ./tests/run_tests.bash
 ```
 
 To run a specific test:
-* Run a container of the image (see the [user guide](#user-guide))
-* Wait until it is configured
+1. Run a container of the image (see the [user guide](#user-guide))
+1. Wait until it is configured
     * The message `[INFO] CNES LEQUAL SonarQube: ready!` is logged.
-* Run a script
+1. Run a script
     ```sh
     $ ./tests/up.bash
     # environnement variables may be modified
-    $ SONARQUBE_ADMIN_PASSWORD="password" SONARQUBE_URL="http://localhost:8080" ./tests/up.bash
+    $ SONARQUBE_ADMIN_PASSWORD="password" SONARQUBE_URL="http://localhost:9000" ./tests/up.bash
     ```
-* Test the exit status of the script with `echo $?`
-    * 0 => success
-    * non-0 => failure
+1. Test the exit status of the script with `echo $?`
+    * zero => success
+    * non-zero => failure
 
 ### How to write tests
 
-Tests are just scripts. To add a test, create a file under the `tests/` folder and make it executable. Then, edit the script. Success and failure are given by exit statuses. A zero exist status is a success. A non-zero exit status is a failure. Note that when using `./tests/start_tests.bash`, only messages on STDERR will by displayed.
+Tests are just scripts. To add a test, create a file under the `tests/` folder and make it executable. Then, edit the script. Success and failure are given by exit statuses. A zero exist status is a success. A non-zero exit status is a failure. Note that when using `./tests/run_tests.bash`, only messages on STDERR will by displayed.
 
 All scripted tests are listed in the [wiki](https://github.com/lequal/sonarqube/wiki#list-of-scripted-integration-tests).
```

Comment:
Maybe you can move this page to `./tests/README.md`, it will be displayed when we browse tests directory.

Reply:
Good idea :+1: